tabs/onboard_logging.js

```diff
@@ -186,11 +175,13 @@ TABS.onboard_logging.initialize = function (callback) {
         for (var i = 0; i < loggingRates.length; i++) {
         	var loggingRate = Math.round(pidRate / loggingRates[i].denom);
         	var loggingRateUnit = " Hz";
-        	if (gcd(loggingRate, 1000)==1000) {
-        		loggingRate /= 1000;
-        		loggingRateUnit = " KHz";
-        		
-        	}
+        	
+            if (loggingRate != Infinity) {
```

Comment:
This seems to be a fix for an unrelated bug. Can you please put this into a separate PR so this can be fixed as soon as possible, while work on the new MSP command is ongoing?